### PR TITLE
pushing modal changes to git

### PR DESCRIPTION
**Pull request by AnnaMikarey**

### CODE REVIEW

**Comment by Lymetta** [comment]
Have we agreed on formatting for FE? If not, then we can disregard the formatting inconsistencies and try merging.

File: client/src/app/disclaimer/disclaimer.component.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { Component } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { ModalComponent } from '../modal/modal.component';
+import { Overlay, OverlayConfig } from '@angular/cdk/overlay';
+import { ComponentPortal } from '@angular/cdk/portal';
```
Comment (AudriusAnusauskas) [inline client/src/app/disclaimer/disclaimer.component.ts:5]: I see the imports that are not used. 
If you are not using them, they should be removed

File: client/src/app/modal/modal.component.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { Overlay, OverlayConfig } from '@angular/cdk/overlay';
+import { ComponentPortal } from '@angular/cdk/portal';
+import { Component, Inject } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { DisclaimerComponent } from '../disclaimer/disclaimer.component';
```
Comment (AudriusAnusauskas) [inline client/src/app/modal/modal.component.ts:5]: same here with the imports

**Review by AudriusAnusauskas — CHANGES_REQUESTED** [review]
Some small adjustments needed

File: client/src/app/disclaimer/disclaimer.component.ts (hunk 1 above)
Comment (AnnaMikarey) [reply client/src/app/disclaimer/disclaimer.component.ts:5]: thanks, fixed it

File: client/src/app/modal/modal.component.ts (hunk 2 above)
Comment (AnnaMikarey) [reply client/src/app/modal/modal.component.ts:5]: thanks, fixed it

**Comment by AudriusAnusauskas** [comment]
So push your updated code :)

**Comment by AnnaMikarey** [comment]
> So push your updated code :)

**Comment by AnnaMikarey** [comment]
fix problem with conflict of branches

**Pull request closed without merging**